scripts/remove

```diff
@@ -7,9 +7,6 @@
 # Source YunoHost helpers
 source /usr/share/yunohost/helpers
 
-# Stop script if errors
-set -u
```

Comment:
Why to remove this

Reply:
I agree for `set -e`. But for `set -u` I think it still important to not execute a function with empty variable.

Reply:
Naively I would think so as well, but turns out that this triggered an issue due to a bug in the core (though could have happened to a bug in the remove script as well) here: https://forum.yunohost.org/t/seafile-installation-failure/14959

- the install fail for some reason
- then the remove script got called
- remove script failed because of undefined `$pid_tail`
- which resulted in the database not being cleaned up properly
- and then reattempting to install failed because of the database already existing